frontend/src/components/dashboard/ProfileBanners.tsx

```diff
@@ -194,26 +198,55 @@ const NoChromeExtensionBanner = () => {
 type NoPremiumBannerProps = {
   runtimeData: RuntimeDataWithPremiumAvailable;
 };
-const NoPremiumBanner = (props: NoPremiumBannerProps) => {
+
+// const NoPremiumBanner = (props: NoPremiumBannerProps) => {
+//   const { l10n } = useLocalization();
+
+//   return (
+//     <Banner
+//       key="premium-banner"
+//       type="promo"
+//       title={l10n.getString("banner-upgrade-headline")}
+//       illustration={<img src={RelayLogo.src} alt="" width={60} height={60} />}
+//       cta={{
+//         target: getPremiumSubscribeLink(props.runtimeData),
+//         content: l10n.getString("banner-upgrade-cta"),
+//         onClick: () => trackPurchaseStart(),
+//         gaViewPing: {
+//           category: "Purchase Button",
+//           label: "profile-banner-promo",
+//         },
+//       }}
+//     >
+//       <p>{l10n.getString("banner-upgrade-copy-2")}</p>
+//     </Banner>
+//   );
+// };
```

Comment:
Obviously no biggie, but a small tip: you don't need to comment out this entire thing either. You can get rid of the ESLint complaint by prepending a single line, accompanied by a comment about why it's disabled:

```js
// Unused but left in for when we no longer want to use <LoyalistPremiumBanner>
// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

VSCode will even tell you what the line is if you hit the :bulb: icon:

![image](https://user-images.githubusercontent.com/4251/171606180-5f53286b-540d-4a23-8cf7-eb8f8c8f196d.png)

Reply:
This is really good to know! May add this in the No Shame doc. I didn't have the VSCode prompts like you do when I hovered over it, is there a library I need to download?